packages/react-components/src/components/styles/VideoGallery.styles.ts

```diff
@@ -14,6 +17,19 @@ export const gridStyle = mergeStyles(videoBaseStyle, {
 
 export const videoTileStyle = {
   root: {
-    borderRadius: '.25rem'
+    borderRadius: theme.effects.roundedCorner4
+  }
+};
+
+export const floatingLocalVideoTileStyle: VideoTileStylesProps = {
+  root: {
+    position: 'absolute',
+    zIndex: 1,
+    bottom: '1rem',
+    right: '1rem',
+    width: '11.25rem',
```

Comment:
Do we have an idea of how we might want thiis local preview tile to scale if we're on really large or small screens?

Reply:
No. In fact, the tile size in figma design is too small and this is a size I decided.